README.md

```diff
@@ -257,6 +265,12 @@ wheel.exclude = []
 # The build tag to use for the wheel. If empty, no build tag is used.
 wheel.build-tag = ""
 
+# Manually specify the wheel tags to use, ignoring other inputs such as
+# ``wheel.py-api``. Each tag must be of the format
+# {interpreter}-{abi}-{platform}. If not specified, these tags are automatically
+# calculated.
+wheel.tags = []
```

Comment:
I don't like exposing this to the pyproject.toml. Maybe we should go ahead and add a way to specify an argument is only supported via the command line. I'd use that for `D` as a shortcut for `cmake.define`.

Edit: this might be useful for overrides, though. Maybe we should just mention that it should not be hard coded except in overrides or something like that.

Reply:
Opened #1078 to handle those. Luckily there was `fail` as the first candidate for that feature.